anago

```diff
@@ -354,8 +354,8 @@ prepare_tree () {
     beta*|official) rev_version_base $label ;;
   esac
 
-  # versionize docs
-  if [[ "$PARENT_BRANCH" == master ]]; then
+  # versionize docs on new branches (from master) only
+  if [[ "$PARENT_BRANCH" == master && "$label" != "alpha" ]]; then
```

Comment:
In this context, this condition is correct.  "alpha" releases only occur on the master.
But for readability, I'll update to condition specifically on the current branch.  Update coming..